trustfall_wasm/src/lib.rs

```diff
@@ -50,6 +51,21 @@ impl Schema {
             .map(Schema::new)
             .map_err(crate::InvalidSchemaError::new)
     }
+
+    pub fn subtypes(&self, type_name: &str) -> Set {
+        let subtypes_iter = match self.0.subtypes(type_name) {
+            Some(iter) => iter,
+            None => panic!("type {type_name} is not part of this schema"),
+        };
```

Comment:
This might be a bit more idiomatic in Rust with `.unwrap_or_else(|| panic!(...))`

Reply:
done in dd7ffda